modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion.tsx

```diff
@@ -35,14 +35,19 @@ export interface SegmentedAccordionSubComponentsInterface {
  * Proptypes for the segmented accordion component.
  */
 export interface SegmentedAccordionPropsInterface extends AccordionProps, IdentifiableComponentInterface,
-    TestableComponentInterface { }
+    TestableComponentInterface { 
+    /**
+     * Type of final view of the list of accordion components.
+     */
+    viewType?: "list-view" | "table-view";
+}
 
 /**
  * Segmented accordion component.
  *
  * @param props - Props injected to the component.
  *
- * @returns Segmented Accordion React Component
+ * @returns
```

Comment:
Can we add what we are returning?

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3528/commits/acf93bbed6a21c13a9d22d84cc8294ea664ad8c4